Default collection install to first path in COLLECTIONS_PATHS (#62870)
ansible/ansible: docs/docsite/rst/user_guide/collections_using.rst

    parent directory is already in a folder called ``ansible_collections``.

> When specifying the install path, use one of the values configured in :ref:`COLLECTIONS_PATHS`, as this is where Ansible itself will expect to

Reviewer: ```suggestion
When using the ``-p`` option to specify the install path, use one of the values configured in :ref:`COLLECTIONS_PATHS`, as this is where Ansible itself will expect to
```

Author: Fixed

> find collections. If no path is specified, a collection will be installed to the first path defined in :ref:`COLLECTIONS_PATHS`, which by

Reviewer: ```suggestion
find collections. If you don't specify a path, ``ansible-galaxy collection`` installs the collection to the first path defined in :ref:`COLLECTIONS_PATHS`, which by
```

Author: Fixed
